app/views/layouts/_footer.html.erb

```diff
@@ -24,3 +24,18 @@
     </div><!-- /.modal-content -->
   </div><!-- /.modal-dialog --> 
 </div><!-- /.modal --> 
+
+<div id="flag-modal" class="modal fade">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h3 class="modal-title">Flag Post for Review</h3>
+      </div>
+
+      <div class="modal-body row">
+        <p>This modal allows you to flag a post for review</p>
+      </div>
+    </div><!-- /.modal-content -->
+  </div><!-- /.modal-dialog --> 
+</div><!-- /.modal --> 
```

Comment:
I know this is a WIP so maybe you planned to change this, but I am wondering if you can find a way to re-use the modal construct for login, since this is not very DRY.  Also, it does not really need to appear on every page, just the topic detail.